src/sentry/static/sentry/app/views/organizationProjectsDashboard/index.jsx

```diff
@@ -60,17 +65,18 @@ class Dashboard extends React.Component {
 
     return (
       <React.Fragment>
-        {favorites.length > 0 && (
-          <TeamSection
-            data-test-id="favorites"
-            orgId={params.orgId}
-            showBorder
-            team={null}
-            title={t('Bookmarked projects')}
-            projects={favorites}
-            access={access}
-          />
-        )}
+        {!hasSentry10 &&
```

Comment:
Why is this only non-sentry10?

Reply:
Right now there's not a guarantee that a user has access to all the bookmarked projects so linking to these could be bad. It happens in the case of org owner or open access orgs